docs/md/melange_completion.md

```diff
@@ -26,25 +26,30 @@ MacOS:
 
 Zsh:
 
+
+```sh
```

Comment:
These docs are autogenerated, please don't modify them.

Reply:
@kaniini ,  since these are auto generated, would it not be better to add them to `.gitignore`?